src/scripts/slurm_submit.sh

```diff
@@ -117,8 +117,9 @@ if [ "$retcode" != "0" ] ; then
 	exit 1
 fi
 
-# The job id is actually the first numbers in the string (slurm support)
-jobID=`echo $jobID | awk 'match($0,/[0-9]+/){print substr($0, RSTART, RLENGTH)}'`
+# The job id is given by the line "Submitted batch job ###".
+# Some installations print additional lines before or after this one.
+jobID=`echo "$jobID" | awk 'match($0,/^Submitted batch job [0-9]+/){print $4}'`
```

Comment:
I guess this is fine, but `match` is really for things that are not used here - it returns the position of the match within the line (or 0 if no match).  Since the match location is not interesting to us here, I think it would be clearer just to put

```suggestion
jobID=`echo "$jobID" | awk '/^Submitted batch job [0-9]+/ {print $4}'`
```

which should have the same behavior.

Reply:
I like the simpler form.